Tasks/ANT/ant.ps1

```diff
@@ -108,9 +108,15 @@ if ($jdkPath)
 }
 
 $buildRootPath = Split-Path $antBuildFile -Parent
-$reportDirectoryName = [guid]::NewGuid()
+$reportDirectoryName = "ReportDirectory"
```

Comment:
add little guid at end like we did in ts implementation, just to make sure we dont end up deleting user folders